### PR TITLE
test ci build

### PR DESCRIPTION
**Pull request by Smarthard**
Testing CI build across forks

### CODE REVIEW

File: .github/workflows/integrate.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ on:
 
 jobs:
   build:
+   # Allow forks to be annotated with code coverage properly
+    permissions:
+      pull-requests: write
```
Comment (MurhafSousli) [inline .github/workflows/integrate.yml:13]: @Smarthard still breaks

Comment (Smarthard) [reply .github/workflows/integrate.yml:13]: изображение

Is it uses workflow from target branch? If so, it won't work until you update that workflow in `master`

Comment (MurhafSousli) [reply .github/workflows/integrate.yml:13]: I don't think it uses the workflow from the master branch

Comment (MurhafSousli) [reply .github/workflows/integrate.yml:13]: I will try something

Comment (Smarthard) [reply .github/workflows/integrate.yml]: try this one, I updated permissions. Maybe it's just mine misunderstanding of where those should belong

Comment (MurhafSousli) [reply .github/workflows/integrate.yml]: I think this should be done from settings https://github.com/orgs/community/discussions/26409#discussioncomment-3251816

Comment (MurhafSousli) [reply .github/workflows/integrate.yml]: I cannot find this settings https://github.com/orgs/community/discussions/26409#discussioncomment-3251816 but I added you as a collaborator

Comment (MurhafSousli) [reply .github/workflows/integrate.yml]: Here actions settings

image
image

Comment (MurhafSousli) [reply .github/workflows/integrate.yml]: Do you want me to merge the yml change into master?
```
# Allow forks to be annotated with code coverage properly
permissions:
  pull-requests: write
  ```

Comment (Smarthard) [reply .github/workflows/integrate.yml]: Pretty much the same. The only difference is in "Fork pull request workflows from outside collaborators" - mine have second option, but it's not changes anything for this case.

I still think it's about workflow config's from target branch or something tho

Comment (Smarthard) [reply .github/workflows/integrate.yml]: I re-run CI with debug option enbaled.

изображение

It fails after trying to fetch this code coverage file

Comment (Smarthard) [reply .github/workflows/integrate.yml]: Btw permissions are look like this in your CI build:

изображение

https://github.com/MurhafSousli/ngx-scrollbar/actions/runs/8816011146/job/24199177925?pr=585

File: .github/workflows/integrate.yml [hunk 2]
```diff
@@ -52,7 +56,7 @@ jobs:
 
       - name: Add Coverage PR Comment
         uses: marocchino/sticky-pull-request-comment@v2
-        if: github.event_name == 'pull_request'
+        if: github.event_name == 'pull_request_target'
```
Comment (MurhafSousli) [inline .github/workflows/integrate.yml:59]: @Smarthard  it looks like `pull_request_target` works

Comment (MurhafSousli) [reply .github/workflows/integrate.yml:59]: Ok, I guess this just prevents executing the task

**Pull request closed without merging**